react-components/src/architecture/concrete/terrain/geometry/RegularGrid2.ts

```diff
@@ -18,7 +18,7 @@ export class RegularGrid2 extends Grid2 {
   public readonly origin: Vector2;
   public readonly increment: Vector2;
 
-  private _buffer: Float32Array; // NaN value in this array means undefined node
+  private _buffer: Float32Array<ArrayBuffer>; // NaN value in this array means undefined node
```

Comment:
I didn't understand how this change is related to the PR, can you elaborate a bit?

Reply:
Yes, I briefly mentioned this in the previous PR: https://github.com/cognitedata/reveal/pull/5074#discussion_r2024961805

Reply:
Ah I see! I think it would be nice to create a task for fixing the typecheck for this package then - it's quite risky to continue working like this.